bin/container_daemon

```diff
@@ -51,6 +65,12 @@ else
   # Unset the swarm key file variable
   unset IPFS_SWARM_KEY_FILE
 
+  # Pre-hook init with scripts
+  init_scripts /docker-prehook.d
+
+  # Post-hook init scripts
+  { sleep 10; init_scripts /docker-posthook.d ; } &
```

Comment:
Separate stage after sleeping for 10 seconds feels brittle. Could not be enough if my node runs on slow CI and takes longer time to init/start etc. 

I feel we don't need a separate directory with posthooks executed after a hardcoded delay.  If anything, we would be waiting for node to start (eg. by probing `/api/v0/id` on the  API port etc).